### PR TITLE
Add Roadmap to Readme

### PR DESCRIPTION
**Pull request by nikmd23**
I've added the roadmap inline into the readme, and made a few other small edits.

### CODE REVIEW

**Comment by nikmd23** [comment]
Thanks @dhaval24.

Whats the process to get this pulled in?

**Comment by dhaval24** [comment]
@nikmd23 the usual process for the code changes is that we wait for atleast 2 reviewers to approve the PR before merging into the master. Since this is readme change and we already discussed I am pulling this in. Please revert later if needed.

File: README.md [hunk 1]
```diff
@@ -28,10 +28,28 @@ After you upgrade, you'll need to merge back any customizations you made to `App
...
+* Download the latest version of [Application Insights Java SDK](https://docs.microsoft.com/en-us/azure/application-insights/app-insights-java-get-started), [scroll down to the getting started section](https://docs.microsoft.com/en-us/azure/application-insights/app-insights-java-get-started) and follow the instructions to manually download the SDK and replace the old `.jar` files.
  
 Compare the old and new `ApplicationInsights.xml`. Many of the changes you see are because we added and removed modules. Reinstate any customizations that you made.
 
+## Application Insights for Java Roadmap
+The Application Insights team have been hard at work to deliver the next wave of features for our Java support and experience. Below is an outline of the features and improvements that are planned for our next release, which is targeted to be completed for mid-year 2018.
```
Comment (grlima) [inline README.md]: This needs to be revised. We will possibly have multiple releases until mid-year 2018. Stable release is days away. After that, the goal is to release whenever there are substantial bug fixes or new features.

File: README.md [hunk 2]
```diff
@@ -28,10 +28,28 @@ After you upgrade, you'll need to merge back any customizations you made to `App
...
+
+### Application Insights for Java SDK 2.0
+In December [we released the beta version of the Application Insights for Java 2.0 SDK](https://github.com/Microsoft/ApplicationInsights-Java/releases/tag/v2.0.0-BETA). That release introduced support for cross-component telemetry correlation and fixed rate sampling. For the final release, we'll be addressing any reliability issues and improving performance.
```
Comment (grlima) [inline README.md]: We're not "improving" performance with stable release. Rather we've conducted perf tests in order to determine the perf overhead of the SDK. As results were satisfactory, we did not make any perf-related code changes.

Comment (dhaval24) [reply README.md]: Yes, this is true. I thought it was implied. If needed we can revert this merge or just file another PR.

File: README.md (hunk 1 above)
Comment (dhaval24) [reply README.md]: Yes, we plan to have a frequent releasing cadence. Infact we would like to get fixes out as soon as we accumulate enough that it quantifies for a minor release.

Comment (nikmd23) [reply README.md]: I'll clarify that this isn't going to all happen in one big-bang 💥 release.

File: README.md (hunk 2 above)
Comment (nikmd23) [reply README.md]: Cool, sounds good.

**Pull request merged**